monai/utils/misc.py

```diff
@@ -584,3 +585,14 @@ def prob2class(x, sigmoid: bool = False, threshold: float = 0.5, **kwargs):
         threshold: threshold value to activate the sigmoid function.
     """
     return torch.argmax(x, **kwargs) if not sigmoid else (x > threshold).int()
+
+
+def path_to_uri(path: PathLike) -> str:
+    """
+    Convert a file path to URI.
+
+    Args:
+        path: input file path to convert, can be a string or `Path` object.
+
+    """
+    return Path(path).as_uri()
```

Comment:
as_uri requires absolute path, I think we run into the issue before, should this `path` be converted to an absolute path?

Reply:
That's a good point, added in the latest commit.

Thanks.